controllers/datafeed_controller.py

```diff
@@ -105,9 +105,13 @@ class FMSAPIEventAlliancesEnqueue(webapp.RequestHandler):
     Handles enqueing getting alliances
     """
     def get(self, when):
+        events = []
```

Comment:
This does not appear to be needed, want me to remove it? (I didn't write the original PR)